_targets/build.project.armv7a9-zynq7000

```diff
@@ -83,10 +85,10 @@ b_build_target() {
 
 
 b_flash_image() {
-	local flash_path="${PREFIX_BOOT}/flash-armv7a9-zynq7000.bin"
+	local flash_path="${PREFIX_BOOT}/flash-armv7a9-zynq7000-${TARGET_PROJECT}.bin"
 
 	rm -f "${flash_path}"
-	dd if="${PREFIX_BOOT}/phoenix-armv7a9-zynq7000.disk" of="${flash_path}" bs=4M 2>/dev/null
+	dd if="${PREFIX_BOOT}/phoenix-armv7a9-zynq7000-${TARGET_PROJECT}.disk" of="${flash_path}" bs=4M 2>/dev/null
```

Comment:
In this case You can use `${TARGET}` instead of `armv7a9-zynq7000-${TARGET_PROJECT}.`